pandas/tests/io/parser/conftest.py

```diff
@@ -101,20 +104,24 @@ def csv1(datapath):
 
 
 @pytest.fixture(params=_all_parsers, ids=_all_parser_ids)
-def all_parsers(request):
+def all_parsers(request, monkeypatch):
     """
     Fixture all of the CSV parsers.
     """
     parser = request.param()
     if parser.engine == "pyarrow":
         pytest.importorskip("pyarrow", VERSIONS["pyarrow"])
-        # Try setting num cpus to 1 to avoid hangs on Azure MacOS/Windows builds
-        # or better yet find a way to disable threads
-        # TODO(GH#44584) pytest.mark.single these tests
```

Comment:
no longer relevant?

Reply:
This shouldn't be necessary if this can run with no threads, but a comment explaining why we are doing this would be helpful